src/code/client.ts

```diff
@@ -634,6 +640,15 @@ class CloudFileManagerClient {
   }
 
   saveFile(stringContent: any, metadata: CloudMetadata, callback: OpenSaveCallback = null) {
+    // if the content didn't change skip the save (but fake it for the client)
+    const savedContentHash = this._computeContentHash(stringContent)
+    if (this.state.contentHash === savedContentHash) {
+      console.log("CFM: File content not changed, skipping sending save to provider!")
```

Comment:
Were you planning to remove this once we're sure it's working correctly?

Reply:
No, I want to leave it in the code in case we get any reports of strange behavior - it will be easier to identify that the save was faked.